CVE-2022-22965/ubuntu1.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: Install Basic Spring Application with CVE-2022-22965 
+  hosts: all
+  become: yes
+  tasks:
+    - name: Update apt package index
+      apt:
+        update_cache: yes
+
+    - name: Ensure Docker is installed
+      apt:
+        name: docker.io
+        state: present
+
+    - name: Install a list of packages
+      ansible.builtin.apt:
+        pkg:
+        - docker-buildx
+
+    - name: Create /opt/spring4shell
+      file:
+        path: /opt/spring4shell
+        state: directory
+
+    - name: Clone spring4shell
+      git:
+        repo: https://github.com/lunasec-io/Spring4Shell-POC.git
```

Comment:
Please add the files from https://github.com/lunasec-io/Spring4Shell-POC to the data folder, do not pull them by URL

Reply:
Ok. I will update that. Thank you!

Reply:
Added the app files to data. Please review